specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/DataFlow.json

```diff
@@ -1,13 +1,24 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "ArtifactsClient",
-    "version": "2019-06-01-preview"
+    "title": "DataFactoryManagementClient",
```

Comment:
pls undo such changes from ArtifactsClient to DataFactoryManagementClient across all files

Reply:
similarly the API version and securityDefinitions portion that got added

Reply:
@ajay-arora please note that this PR is only for diff. I will need ADF team to help update the Swagger spec rather than do it by myself.